backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/service/OrgChangeService.java

```diff
@@ -78,4 +85,21 @@ public boolean hasAnyMembershipProduct(final OrgId orgId)
 	{
 		return orgChangeRepo.hasAnyMembershipProduct(orgId);
 	}
+
+	public GroupCategoryId getTargetGroupCategoryId(final GroupCategoryId sourceGroupTemplateId, final OrgId targetOrgId)
```

Comment:
why not something like `boolean isGroupTemplateContainsProductsInTargetOrg(... same args ...)` ?

Reply:
Done. But it's ``isGroupCategoryContainsProductsInTargetOrg``.
I only check if it there is a product which has the same group category in the target org.

Reply:
I also renamed the parameter to avoid confusion. Because category and template are not the same thing.